packages/vats/src/core/manifest.js

```diff
@@ -218,6 +219,11 @@ const SHARED_CHAIN_BOOTSTRAP_MANIFEST = harden({
       vattp: true,
     },
   },
+  [noProvisioner.name]: {
+    produce: {
+      provisioning: 'provisioning',
+    },
+  },
```

Comment:
Putting both `makeProvisioner` and `noProvisioner` in the same manifest would have them race against each other to produce `provisioning`.

I think the right thing is to remove this `noProvisioner.name` property.

Reply:
Oh I misunderstood the manifest I think!